systemd/macros.jinja

```diff
@@ -1,58 +1,83 @@
-{%- macro files_switch(prefix,
-                       files,
+{%- macro files_switch(files,
                        default_files_switch=['id', 'os_family'],
                        indent_width=6) %}
-  {#
+  {#-
     Returns a valid value for the "source" parameter of a "file.managed"
     state function. This makes easier the usage of the Template Override and
     Files Switch (TOFS) pattern.
 
     Params:
       * prefix: pillar prefix to custom ':files_switch'. Colons ':'
-        are replaced by '/' to be used as directory prefix
-      * files: ordered list of files to look for, with full path
-      * default_files_switch: if there's no pillar 'prefix:files_switch'
-        this is the ordered list of grains to use as selector switch of the
-        directories under "prefix/files"
+        are replaced by '/' to be used as directory prefix (<path_prefix>)
+      * files: ordered list of files to look for
+      * default_files_switch: if there's no pillar
+        '<prefix>:files_switch' this is the ordered list of grains to
+        use as selector switch of the directories under
+        "<path_prefix>/files"
       * indent_witdh: indentation of the result value to conform to YAML
 
     Example:
 
     If we have a state:
 
-      /etc/xxx/xxx.conf:
-        file:
-          - managed
-          - source: {{ files_switch('xxx', ['/etc/xxx/xxx.conf',
-                                            '/etc/xxx/xxx.conf.jinja']) }}
+      Deploy configuration:
+        file.managed:
+          - name: /etc/yyy/zzz.conf
+          - source: {{ files_switch('xxx', ['/etc/yyy/zzz.conf',
+                                            '/etc/yyy/zzz.conf.jinja']) }}
           - template: jinja
 
     In a minion with id=theminion and os_family=RedHat, it's going to be
     rendered as:
 
-      /etc/xxx/xxx.conf:
-        file:
-          - managed
+      Deploy configuration:
+        file.managed:
+          - name: /etc/yyy/zzz.conf
           - source:
-            - salt://xxx/files/theminion/etc/xxx/xxx.conf
-            - salt://xxx/files/theminion/etc/xxx/xxx.conf.jinja
-            - salt://xxx/files/RedHat/etc/xxx/xxx.conf
-            - salt://xxx/files/RedHat/etc/xxx/xxx.conf.jinja
-            - salt://xxx/files/default/etc/xxx/xxx.conf
-            - salt://xxx/files/default/etc/xxx/xxx.conf.jinja
+            - salt://xxx/files/theminion/etc/yyy/zzz.conf
+            - salt://xxx/files/theminion/etc/yyy/zzz.conf.jinja
+            - salt://xxx/files/RedHat/etc/yyy/zzz.conf
+            - salt://xxx/files/RedHat/etc/yyy/zzz.conf.jinja
+            - salt://xxx/files/default/etc/yyy/zzz.conf
+            - salt://xxx/files/default/etc/yyy/zzz.conf.jinja
+          - template: jinja
```

Comment:
All of these comment changes have come from `template-formula`.  These still have to be updated everywhere.